### PR TITLE
RU: Translation for guide/eager.ipynb 

### PR DESCRIPTION
**Pull request by stabuev**

### CODE REVIEW

File: site/ru/guide/eager.ipynb [hunk 1]
```diff
@@ -0,0 +1,1376 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vkHs32GqweYS"
+      },
+      "source": [
+        "### Обучение модели\n",
+        "\n",
+        "В следующем примере создается многослойная модель которая классифицирует стандартные\n",
```
Comment (Alek-dr) [inline site/ru/guide/eager.ipynb:438]: модель, которая классифицирует

File: site/ru/guide/eager.ipynb [hunk 2]
```diff
@@ -0,0 +1,1376 @@
...
+        "рукописные цифры MNIST. Он демонстрирует оптимизатор и API слоев для построения\n",
```
Comment (Alek-dr) [inline site/ru/guide/eager.ipynb:439]: Пример демонстрирует / в примере демонстрируется

File: site/ru/guide/eager.ipynb [hunk 3]
```diff
@@ -0,0 +1,1376 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "kKpOlHPLqEgl"
+      },
+      "source": [
+        "### Переменные и оптимизаторы\n",
+        "\n",
+        "Объекты `tf.Variable` хранят изменяемые значения типа `tf.Tensor` доступные во время\n",
```
Comment (Alek-dr) [inline site/ru/guide/eager.ipynb:629]: Объекты `tf.Variable` хранят изменяемые значения типа `tf.Tensor`, доступные во время обучения, чтобы упростить автоматическое дифференцирование

File: site/ru/guide/eager.ipynb [hunk 4]
```diff
@@ -0,0 +1,1376 @@
...
+      "metadata": {
+        "colab_type": "text",
+        "id": "Q7x1CDurl3IG"
+      },
+      "source": [
+        "Далее:\n",
+        "\n",
+        "1. Создание модели.\n",
+        "2. Производные функции потерь относительно параметров модели.\n",
+        "3. Стратегия обновления переменных основанная на производных."
```
Comment (Alek-dr) [inline site/ru/guide/eager.ipynb:701]: Стратегия обновления переменных, основанная на производных

File: site/ru/guide/eager.ipynb [hunk 5]
```diff
@@ -0,0 +1,1376 @@
...
+        "colab": {},
+        "colab_type": "code",
+        "id": "SbXJk0f2lztg"
+      },
+      "outputs": [],
+      "source": [
+        "model = Linear()\n",
+        "optimizer = tf.keras.optimizers.SGD(learning_rate=0.01)\n",
+        "\n",
+        "print(\"Начальные потери: {:.3f}\".format(loss(model, training_inputs, training_outputs)))\n",
```
Comment (Alek-dr) [inline site/ru/guide/eager.ipynb:717]: Я бы не переводил Initial loss, loss at step, но это дело вкуса

File: site/ru/guide/eager.ipynb [hunk 6]
```diff
@@ -0,0 +1,1376 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "gieGOf_DqEhK"
+      },
+      "source": [
+        "### Кастомные градиенты\n",
+        "\n",
+        "Кастомные градиенты это простой способ переписать градиенты. В функции прямого прохода, определите градиент относительно\n",
+        "входных, выходных или промежуточных результатов. For example, here's an easy way to clip\n",
```
Comment (Alek-dr) [inline site/ru/guide/eager.ipynb:1072]: Например, вот легкий способ обрезать градиенты в обратном распространении:

File: site/ru/guide/eager.ipynb [hunk 7]
```diff
@@ -0,0 +1,1376 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "EGjDcGxIqEfX"
+      },
+      "source": [
+        "\n",
+        "\n",
+        "TensorFlow eager execution это императивная программная среда, которая\n",
```
Comment (ilyaspiridonov) [inline site/ru/guide/eager.ipynb:79]: пунктуация:
TensorFlow eager execution — это

File: site/ru/guide/eager.ipynb [hunk 8]
```diff
@@ -0,0 +1,1376 @@
...
+        "вычисляет операции немедленно, без построения графов: операции возвращают\n",
+        "конкретные значения вместо построения вычислительного графа для последующего запуска. Это\n",
+        "облегчает начало работы с TensorFlow и отладкой моделей, а также\n",
+        "шаблонный код. Чтобы следовать этому руководству, выполните приведенныее ниже примеры кода\n",
+        "в интерактивном интерпретаторе `python`.\n",
+        "\n",
+        "Eager execution это гибкая платформа машинного обучения для исследований и\n",
```
Comment (ilyaspiridonov) [inline site/ru/guide/eager.ipynb:86]: Eager execution — это

File: site/ru/guide/eager.ipynb [hunk 9]
```diff
@@ -0,0 +1,1376 @@
...
+        "  структуры данных Python. Быстро итерируйте по небольшим моделям и данным.\n",
+        "* *Более простая отладка*—Вызывайте операции напрямую, чтобы проверять работающие модели и тестируйте\n",
+        "  изменения. Используйте стандартные инструменты Python для немедленных сообщений об ошибках.\n",
+        "* *Естественный порядок выполнения*—Используйте порядок выполнения Python вместо порядка выполнения\n",
+        "  графа, упрощая спецификации динамических моделей.\n",
+        "\n",
+        "Eager execution поддерживает большинство операций TensorFlow акселерацию GPU.\n",
+        "\n",
+        "Замечание: Некоторые модели могут испытывать повышенную нагрузку при включенном eager\n",
+        "execution. Улучшения производительности продолжаются, но пожалуйста\n",
```
Comment (ilyaspiridonov) [inline site/ru/guide/eager.ipynb:99]: Улучшения производительности продолжаются -> Мы продолжаем работать над улучшением производительности

File: site/ru/guide/eager.ipynb [hunk 10]
```diff
@@ -0,0 +1,1376 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "8huKpuuAwICq"
+      },
+      "source": [
+        "## Eager обучение"
```
Comment (ilyaspiridonov) [inline site/ru/guide/eager.ipynb:379]: Режим обучения eager training?

File: site/ru/guide/eager.ipynb [hunk 11]
```diff
@@ -0,0 +1,1376 @@
...
+        "colab_type": "text",
+        "id": "mp2lCCZYrxHd"
+      },
+      "source": [
+        "### Вычисление градиентов\n",
+        "\n",
+        "[Автоматическое дифференцирование](https://en.wikipedia.org/wiki/Automatic_differentiation)\n",
+        "полезно для реализации алгоритмов машинного обучения, таких как\n",
+        "[обратное распространение](https://en.wikipedia.org/wiki/Backpropagation) для обучения\n",
+        "нейронных сетей. Во время eager execution, используй `tf.GradientTape` чтобы отслеживать\n",
```
Comment (ilyaspiridonov) [inline site/ru/guide/eager.ipynb:394]: используй -> используйте

File: site/ru/guide/eager.ipynb (hunk 1 above)
Comment (stabuev) [reply site/ru/guide/eager.ipynb:438]: спасибо, запятые всегда были моим слабым местом :)

**Comment by stabuev** [comment]
@Alek-dr, @ilyaspiridonov Thank you for the valuable reviews. All your suggestions were added to the translation :).
